src/server/ApiRoutes/Hold.js

```diff
@@ -269,6 +269,8 @@ function confirmRequestServer(req, res, next) {
         );
       }
 
+      res.status(404).redirect(`${appConfig.baseUrl}/404`);
```

Comment:
Given that this is 50 lines south of the patronId conditional, could we either 1) add an inline comment explaining why we're 404ing here or (my pref) 2) move the 404 redirect way up to the top inside a `if (patronIdFromHoldRequest !== patronId) {` conditional so that that it's clear why we'd want to 404 and that whole 50 line fetchBib block can safely follow it without having to be inside a conditional? That make sense? I feel like any time we can get one simple conditional out of the way, we should do that first.

Reply:
Good idea! I've pushed that change